html/modules/custom/reliefweb_import/src/Service/WorkDayJobImporter.php

```diff
@@ -31,12 +31,53 @@ public function setSettings(array $settings): void {
     $this->settings = $settings;
   }
 
+  /**
+   * Validate settings.
+   */
+  public function validateSettings(): void {
+    $required_settings = [
+      'base_url',
+      'tenant',
+      'client_id',
+      'client_secret',
+      'refresh_token',
+      'source_id',
+      'uid',
+    ];
+
+    $missing_settings = [];
+    foreach ($required_settings as $setting) {
+      if (empty($this->settings[$setting])) {
```

Comment:
This would catch settings with a possible valid "FALSE" or "0" value.

Reply:
All of them should have a "real" value